src/main/java/uk/co/ramp/policy/alert/AlertPolicyContext.java

```diff
@@ -0,0 +1,44 @@
+package uk.co.ramp.policy.alert;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.Reader;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.boot.SpringBootConfiguration;
+import org.springframework.context.annotation.Bean;
+import uk.co.ramp.ConfigurationException;
+import uk.co.ramp.Population;
+import uk.co.ramp.event.CompletionEventListGroup;
+
+@SpringBootConfiguration
+public class AlertPolicyContext {
+  private static final String ALERT_SETTINGS_LOCATION = "input/alertPolicies.json";
```

Comment:
Not sure if we should be keeping these policies hard coded now we have an inputs file? Unless you think this should be here because it is context specific? I think the same goes for the isolation inputs.

Reply:
Yep agreed, both should move over to the inputs file. I'll update this